cmd/conformance-tests/runner.go

```diff
@@ -1346,7 +1346,8 @@ func supportsStorage(cluster *kubermaticv1.Cluster) bool {
 func supportsLBs(cluster *kubermaticv1.Cluster) bool {
 	return cluster.Spec.Cloud.Azure != nil ||
 		cluster.Spec.Cloud.AWS != nil ||
-		cluster.Spec.Cloud.GCP != nil
+		cluster.Spec.Cloud.GCP != nil ||
+		(cluster.Spec.Cloud.Hetzner != nil && cluster.Spec.Version.Minor() >= 18)
```

Comment:
Hetzner CCM is documented to be for 1.18+ -- older versions of the CCM might work for older Kubernetes versions, but this is for now safe enough IMHO.

Reply:
I'd like to suggest to change the body of this function to switch. Will make it more readable.

Reply:
I honestly don't see the improvement a switch statement would bring in Go, where either all branches have to return or have an explicit fallthrough. This however is a nice, concise disjunction. :)